File: mario.py
from cs50 import get_int

while True:
    n = get_int("Integer height: ")
    if n >= 0 and n <= 23:
        break

# Print out this many rows
for i in range(n):
    # Print out this many column
    for j in range(n - i - 1):
        print(" ", end="")
    for k in range(i + 2):
        print("#", end="")
    print()

File: mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
    int n;
    do
    {
        n = get_int("Integer height: ");
    }
    while (n < 0 || n > 23);
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 2; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}